File: trabalho 1/VitorBasso11611BCC034/programas tarefa 1/java/micro/micro09.java
public class micro09{
    public static void main(String[] args){
        double preco, venda, novo_preco;
        novo_preco = 0;
        System.out.print("Digite o preco: ");
        preco = Double.parseDouble(System.console().readLine());
        System.out.print("Digite a venda: ");
        venda = Double.parseDouble(System.console().readLine());
        if((venda < 500) || (preco < 30)){
            novo_preco = preco + (10/100) * preco;
        }else if(((venda >= 500) && venda < 1200) || ((preco >= 30) && preco >= 80)){
            novo_preco = preco + (15/100) * preco;
        }else if ((venda >= 1200) || (preco >=80)){
            novo_preco = preco - (20/100) * preco;
        }
        System.out.println("O novo preco e " + novo_preco);
    }
}

File: trabalho 4/codigos/lua/micro/micro09.lua
function int main()
    int preco, venda, novo_preco
    print("Digite o preco: ")
    preco = io.read('*n')
    print("Digite a venda: ")
    venda = io.read('*n')
    if venda < 500 or preco < 30 then
        novo_preco = preco + 10 / 100 * preco
    else if (venda >= 500 and venda <= 1200) or (preco >=30 and preco <80) then
        novo_preco = preco + 15 / 100 * preco
    else if venda >= 1200 or preco >= 80 then
        novo_preco = preco - 20 / 100 * preco
    end
    end
    end
    print("O novo preco e ")
    print(novo_preco)
    print("\n")
    return 1
end    

main()
